src/amo/pages/Addon/index.js

```diff
@@ -299,6 +299,27 @@ export class AddonBase extends React.Component {
     ) : null;
   }
 
+  renderDevCommentsCard = () => {
+    const { addon, i18n } = this.props;
+
+    if (!addon || !addon.developer_comments) return null;
+
+    const devComments = sanitizeUserHTML(addon.developer_comments);
+    const showMoreCardName = 'Addon-developer-comments';
+
+    /* eslint-disable react/no-danger */
+    return (
+      <ShowMoreCard
+        className={showMoreCardName}
+        header={i18n.gettext(`Developer's comments`)}
```

Comment:
@jvillalobos is this header title good? Or should we use `Author`? Or even `Developer`/`Artist`/`Author` depending on the context. What we should have when there are more than author?

Reply:
I think `Developer comments` without the possessive is better because it's more generic (in English, at least).